crates/kubelet/src/config.rs

```diff
@@ -66,6 +68,8 @@ pub struct Config {
     /// device plugins lives. This is also where device plugins
     /// should host their services.
     pub device_plugins_dir: PathBuf,
+    /// Wasm runtime to use to execute your wasm
+    pub provider: Provider,
```

Comment:
So generally speaking we've avoided adding internal implementation details like a `Provider` type directly into the `kubelet` crate. The goal of this crate is to implement the kubelet API without leaking any implementation details of Krustlet. That way clients can use the `kubelet` crate to interact with the Kubernetes API and impersonate a kubelet any way they choose.

Since you're just introducing this type to check a string value.... This seems more appropriate in the provider code itself.